src/configuration/default_configuration.ts

```diff
@@ -0,0 +1,25 @@
+import { IConfiguration } from './types'
+
+export const DEFAULT_CONFIGURATION: IConfiguration = {
```

Comment:
This has been effectively refactored out of `ArgvParser`, because that's no longer the only way we get config.